.travis.yml

```diff
@@ -1,18 +1,17 @@
 language: python
 
 python:
-  - 2.7
-  - 3.5
+  - 3.8
 
 matrix:
   allow_failures:
-  - python: 3.5
+  - python: 3.8
 
 before_install:
   - sudo apt-get -qq update
   - sudo apt-get install -y libboost-python-dev gfortran
   - pip install setuptools numpy scipy astropy
 
-install: "python setup.py install"
+install: "pip install ."
```

Comment:
It may be necessary to update `pip` before using it. Not sure how this should be done in Travis. The command would (obviously) be something like `python3 -m pip install -U pip`